### PR TITLE
Finish category index and front page

### PR DESCRIPTION
**Pull request by sidewinder2020**
LIST OF CHANGES:
- login and total price of cart are on the right side now
- flash message added for incorrect login attempt
- styled the front page, messages, and category index
-

THINGS I'M UNSURE ON:
- it's beautiful, BUT make the login or register a button? (on the cart page)
-
-

RSPEC GREEN?
YES
NO- WHY?

YES

*MENTION GROUP MEMBER TO REVIEW PR*

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ def create
       session[:logged_in?] = true
       redirect_to dashboard_path
     else
+      flash[:cannot_login] = "You have entered an incorrect username or password."
```
Comment (snayrouz) [inline app/controllers/sessions_controller.rb:17]: sweet thanks

File: app/views/carts/show.html.erb [hunk 2]
```diff
@@ -27,13 +27,13 @@
...
 </table>
-    <p>Total Cart Price:<%= number_to_currency(@cart.cart_total) %></p>
+    <p class="text-right">Total Cart Price:<%= number_to_currency(@cart.cart_total) %></p>
 </div>
 <div class="container-fluid">
     <% if current_user %>
     <%= button_to "Checkout", new_order_path(current_user), method: :get %>
   <% else %>
-    <%= link_to "Login or Register to checkout", login_path %> <br />
+    <p class="text-right"><%= link_to "Login or Register to checkout", login_path %></p><br />
```
Comment (snayrouz) [inline app/views/carts/show.html.erb:36]: righty tighty

**Comment by ButeraMV** [comment]
fancy!

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (blsrofe) [reply app/controllers/sessions_controller.rb:17]: Nice!

**Pull request merged**